python/get_table_list.py

```diff
@@ -0,0 +1,21 @@
+import keys, slope_api
+
+def get_table_list():
+    api = slope_api.SlopeApi()
+    api.authorize(keys.api_key, keys.api_secret)
+
+    # Get a List of all Data Tables on a Model
+    tables = api.list_data_tables(11759)

```

Comment:
Consider pulling all of these parameters out into constants or variables, with descriptive names, to help users understand what the parameters represent without navigating to the API class.